website/source/docs/configuration/seal/gcpckms.html.md

```diff
@@ -35,6 +35,21 @@ seal "gcpckms" {
 }
 ```
 
+## Setting the Google Compute Instance Scope
+If you wish to use auto-unseal without having a credentials file on the instance, you can set the cloudkms scope on your instance.  Here is an example command:
+
+```
+gcloud beta compute instances set-scopes myinstance --zone us-east1-b --scopes=https://www.googleapis.com/auth/cloudkms
+```
+
+If you're using Terraform to configure your instance, your service account block may look something like this:
```

Comment:
We don't really go into multiple tools in this section.  This may be more appropriate in an Auto Unseal guide that is more multi-tool.  Can we just keep it native tools like the gcloud CLI above?

Reply:
I'm ok with whatever you all decide.  Just wanted to make sure this was documented somewhere.